sql/src/influxql/planner.rs

```diff
@@ -38,20 +103,29 @@ impl<'a, P: MetaProvider> Planner<'a, P> {
     /// and build plan then.
     pub fn statement_to_plan(self, stmt: InfluxqlStatement) -> Result<Plan> {
         match stmt {
-            InfluxqlStatement::Select(_) => self.select_to_plan(stmt),
+            // TODO: show measurement is a temp workaround, it should be implemented in influxql
+            // crates.
             InfluxqlStatement::ShowMeasurements(stmt) => self.show_measurements_to_plan(*stmt),
-            InfluxqlStatement::CreateDatabase(_)
-            | InfluxqlStatement::ShowDatabases(_)
-            | InfluxqlStatement::ShowRetentionPolicies(_)
-            | InfluxqlStatement::ShowTagKeys(_)
-            | InfluxqlStatement::ShowTagValues(_)
-            | InfluxqlStatement::ShowFieldKeys(_)
-            | InfluxqlStatement::Delete(_)
-            | InfluxqlStatement::DropMeasurement(_)
-            | InfluxqlStatement::Explain(_) => Unimplemented {
-                msg: stmt.to_string(),
+            _ => {
```

Comment:
I think the situation may be same with what in sql ? 
For example `CreateDatabase` statement(like `CreateTable` in sql part), it may be deep coupling with iox and it is too hard to reuse?